e2e/fixtures/electron-app.ts

```
@@ -58,74 +55,74 @@ function createTestDataDir(): string {
-
-    // Wait for the app to be ready (DOM loaded)
-    await window.waitForLoadState('domcontentloaded');
-
-    // Give the app a moment to initialize React
-    await window.waitForTimeout(500);
-
-    await use(window);
-  },
+	// Test data directory for isolation
+	// eslint-disable-next-line no-empty-pattern
+	testDataDir: async ({}, use) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)